src/components/SearchCard/SearchCardBody.js

```diff
@@ -0,0 +1,67 @@
+import { Box, Heading, Paragraph, Text } from 'grommet'
+import { Link } from 'components/shared/Link'
+import { formatString } from 'helpers/formatString'
+
+export const SearchCardBody = ({
+  alternateAccessionCode = '',
+  description = '',
+  publicationTitle = '',
+  sampleMetadataFields = []
+}) => {
+  return (
+    <Box pad={{ top: 'medium', bottom: 'small' }}>
+      <Box>
+        <Heading level={4} weight="500">
+          Description
+        </Heading>
+        {description ? (
```

Comment:
This logic is repeated a lot, could you simplify it by writing a component?

Reply:
Added as the sub-component `CardBlock` to render each section of `SearchCardBody`, thank you!

**NOTE:** There is a shared component called [`TextNull`](https://github.com/AlexsLemonade/refinebio-web/blob/ad6977ccf45d49eb1684c0cf4a97ee06140fd0a4/src/components/shared/TextNull.js) for texts with null values was created in the PR [36 - Implement Samples Table](https://github.com/AlexsLemonade/refinebio-web/pull/41).